sonar-plugin-server/src/main/java/jetbrains/buildserver/sonarplugin/sqrunner/ServerManagementProjectTab.java

```diff
@@ -77,4 +86,8 @@ public void fillModel(@NotNull final Map<String, Object> model, @NotNull final H
         }
         return infoMap;
     }
+
+    private boolean userHasPermission() {
+        return AuthUtil.hasGlobalPermission(securityContext.getAuthorityHolder(), Permission.CHANGE_SERVER_SETTINGS);
```

Comment:
Same as in ManageSQSActionController:88 - `EDIT_PROJECT` permission should be used